playlist_generator/Nearest_Neighbours.py

```diff
@@ -81,26 +101,3 @@ def find_songs_by_valence(genre, valence, n=10):
     data = data.sort_values(['diff', 'popularity'], ascending=[True, False])
     songs = data.head(n)
     return songs
-
-
-if __name__ == '__main__':
```

Comment:
I think you could leave this here tbh, will only ever run with the command `python Nearest_Neighbours.py`

Reply:
Haha I changed the behaviour of the functions so this code wouldn't work anymore. I get what you mean though.

Reply:
Yea no worries, can just remove it, I was just wondering haha